include/deal.II/grid/grid_generator.h

```diff
@@ -304,6 +304,49 @@ namespace GridGenerator
   cheese(Triangulation<dim, spacedim> &   tria,
          const std::vector<unsigned int> &holes);
 
+  /**
+   * \brief Plate with a hole.
+   *
+   * Generate a plate with a hole. The geometry consists of 2 parts:
+   * First, a square part with size @p outer_radius and a hole of radius @p inner_radius .
+   * Cells in this part will have TransfiniteInterpolationManifold with id
+   * @p tfi_manifold_id attached. Additionally, faces at the hole will use
+   * PolarManifold (in 2D) or CylindricalManifold (in 3D). The center of this
+   * part can be prescribed via @p center .
+   * Second, the remaning bulk part which is specified via padding
+   * parameters @p pad_bottom, @p padding_top, @p padding_left and @padding_right.
```

Comment:
As its not immediately apparent to me from the description, I think that it could be made a little more clear as to what the resulting geometry is going to look like. Perhaps you should state what the final length and height of the geometry are going to be. If I understand correctly, then `length = padding_top + outer_radius + padding_bottom` and `width = padding_left + outer_radius + padding_right`.